VotingApplication/VotingApplication.Web/Scripts/Controllers/Create/ConfirmStepController.js

```diff
@@ -17,6 +20,11 @@
         $scope.styleIsValid = styleIsValid;
         $scope.expiryIsValid = expiryIsValid;
         $scope.invitationsAreValid = invitationsAreValid;
+        $scope.formatUtcPollExpiry = formatUtcPollExpiry;
+        $scope.expiryDateIsInPast = expiryDateIsInPast;
+        $scope.createPoll = createPoll;
+
+        var startingDate = new Date();
```

Comment:
Moment's interaction with angular is a bit flakey and can cause digest problems. Not 100% sure why, but simply using Date objects and wrapping them in moment when we convert to UTC seems to fix the problem.